functions-python/reverse_geolocation_populate/function_config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "reverse-geolocation-populate",
+  "description": "Populate the database with reverse geolocation data",
+  "entry_point": "reverse_geolocation_populate",
+  "timeout": 3600,
+  "memory": "8Gi",
```

Comment:
Should we remove the `memory` field? I think we are using `memory` and `available_memory` for the same purpose in different functions.